app.rb

```diff
@@ -41,6 +41,10 @@
     success_response = false
   end
 
+  if params[:raw]
+    return facts
+  end
```

Comment:
```suggestion
  return facts if params[:raw] == 'true'
```

Reply:
We need to change the `content_type 'application/json', 'charset' => 'utf-8'` to `content_type 'text/plain', 'charset' => 'utf-8'` when the parameter is also raw in order to avoid responding with an invalid JSON header.